futures-util/src/stream/futures_unordered/iter_mut.rs

```diff
@@ -10,7 +10,7 @@ pub struct IterMut<'a, F: 'a> {
     pub(super) _marker: PhantomData<&'a mut FuturesUnordered<F>>
```

Comment:
Can you add an `Unpin` bound to the `IterMut` type itself, too?

Reply:
Done